docs/DeveloperGuide.adoc

```diff
@@ -337,10 +387,121 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. SML shows an error message.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Add new phone (UC02)
+
+*MSS*
+
+1. Actor input phone name and quantity.
+2. System adds phone to the list of phones.
+3. System shows the new list of phones.
+
+*Extensions*
+
+[none]
+* 1a. Actor inputs wrong date format.
++
+Use case ends.
+[none]
+** 1a1. SML requests for correct data.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Check quantity of phone (UC03)
+
+*MSS*
+
+1. Actor requests to check stock of a phone.
+2. System displays the stock status for a phone.
+
+*Extensions*
+
+[none]
+* 1a. System is unable to find the phone as it does not exist.
++
+Use case ends.
+[none]
+** 1a1. System shows error message that phone does not exist.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Confirm order (UC04)
+
+*MSS*
+
+1. Actor requests to check stock of a phone (UC03).
+2. Actor add order to the system. System displays the new order list.
+3. Actor schedule order. (UC05)
+4. Actor mark order as completed. System displays completed order message.
```

Comment:
I believe each step in a use case should only describe one action, so the actions taken by the system should be a new step in the use case.